modules/navier_stokes/doc/intro_slides/slides/index.md

```diff
@@ -0,0 +1,227 @@
+# The Navier-Stokes Module
+
+!style halign=center fontsize=120%
+!datetime today format=%B %Y
+
+!---
+
+# Overview
+
+Main capabilities:
+
+- Incompressible fluid flows
+- Weakly-compressible fluid flows
+- Compressible fluid flows
+- Free flows and flows in porous media describing homogenized structures
+
+Utilizies the following techniques:
+
+- Continuous Finite Element Discretization (maintained, but not intensely developed)
+- Finite Volume Discretization (current development direction)
+
+!---
+
+# The Navier-Stokes equations ([!citet](radman2021development))
+  style=font-size:26pt
+
+!style! fontsize=80%
+
+- Conservation of +mass+:
+
+!equation id=mass-eq
+\frac{\partial \gamma \rho}{\partial t} + \nabla \cdot \left( \rho \vec{u} \right)  = 0
+
+- Conservation of (linear) +momentum+:
+
+!equation id=momentum-eq
+\frac{\partial \rho  \vec{u}}{\partial t} + \nabla \cdot \left(\gamma^{-1}  \rho \vec{u} \otimes \vec{u}\right)
+= \nabla \cdot \left(\gamma \mu_\text{eff} \left(\nabla\vec{u}_I + \left(\nabla \vec{u}_I\right)^T-\frac{2}{3}\nabla\cdot\vec{u}_I\mathbb{I}\right)\right) -\gamma\nabla p + \gamma  \rho \vec{g} -  W  \vec{u}_I
+
+!style-end!
+
+!row!
+!col! width=30%
+
+!style! fontsize=70%
+- $\gamma$ - porosity
+- $\vec{u}_I$ - real (or interstitial) velocity
+- $\vec{u} \equiv \gamma \vec{u}_I$ - superficial velocity
+!style-end!
+
+!col-end!
+
+!col! width=60%
+
+!style! fontsize=70%
+- $\rho = \rho(p,T)$ - density
+- $\mu_{eff}$ - effective dynamic viscosity (laminar + turbulent)
+- $p$ - pressure
+- $W$ - friction tensor (using correlations)
+!style-end!
+
+!col-end!
+!row-end!
+
+!style! fontsize=80%
+- If $\gamma=1$ then $\vec{u}=\vec{u}_I$ and we get the original Navier-Stokes equations back.
+!style-end!
+
+
+!---
+
+# The Navier-Stokes equations (contd.)
+  style=font-size:26pt
+
+- These are supplemented by the conservarion of +energy+:
+
+  +Assumptions:+ Weakly-compressible fluid, kinteic energy of the fluid can be neglected
+
+!equation id=enthalpy-eq
+\frac{\partial \gamma \rho c_p T}{\partial t} + \nabla \cdot \left(  \rho c_p T \vec{u}\right)
+= \nabla \cdot \left(\kappa_f \nabla T \right) - \alpha (T - T_s) + S_{ext}
+
+!row!
+!col! width=30%
+
+!style! fontsize=80%
+- $c_p$ - specific heat
+- $T$ - fluid temperature
+- $\kappa_f$ - effective thermal conductivity
+!style-end!
+
+!col-end!
+
+!col! width=60%
+
+!style! fontsize=80%
+- $\alpha$ - volumetric heat-transfer coefficient (using correlations)
+- $T_s$ - solid temperature
+- $S_{ext}$ - volumetric source term
+!style-end!
+
+!col-end!
+
+!row-end!
+
+!---
+
+# The Navier-Stokes equations (contd.)
+  style=font-size:26pt
+
+The equations are extended with initial and boundary conditions:
+
+- Few examples:
+
+  - Inlet: $\vec{u}=\vec{u}_{inlet}, \quad T=T_{inlet}, \quad \vec{r} \in \partial\Omega_{inlet}$
```

Comment:
Velocity Inlet
Pressure Outlet

etc ?
since we can do different inlets (mass flux inlet, pressure inlet etc)

Reply:
Good idea!